GOVERNANCE.md

```diff
@@ -0,0 +1,26 @@
+# restic project governance
+
+## Overview
+
+The restic project uses a governance model commonly described as Benevolent
+Dictator For Life (BDFL). This document outlines our understanding of what this
+means. It is derived from the [i3 window manager project
+governance](https://raw.githubusercontent.com/i3/i3/next/.github/GOVERNANCE.md).
+
+## Roles
+
+* user: anyone who interacts with the restic project
+* core contributor: a handful of people who have contributed significantly to
+  the project by any means (issue triage, support, documentation, code, etc.).
+  Core contributors are recognizable via GitHub’s "Member" badge.
+* Benevolent Dictator For Life (BDFL): a single individual who makes decisions
+  when consensus cannot be reached. restic's current BDFL is [@fd0](https://github.com/fd0).
+
+## Decision making process
+
+In general, we try to reach consensus in discussions. In case consensus cannot
+be reached, the BDFL makes a decision.
+
+## Contribution process
+
+Please see [CONTRIBUTING](CONTRIBUTING.md).
```

Comment:
Maybe add a section how the CoC would be inforced and by whom? (source material e.g. here: https://opensource.guide/code-of-conduct/#enforcing-your-code-of-conduct)

Reply:
Hm. The Django enforcement manual feels a bit too much.

Reply:
I've added a section, please let me know what you think

Reply:
LGTM